app/installer.sh

```diff
@@ -107,6 +110,53 @@ if [ "$OS" = "Linux" ]; then
     printf "INFO\tDistribution: %s\n" "$DISTRO"
 fi
 
+# Sets `VERSION` to the git tag representing the latest preview version.
+set_preview_version() {
+  if [ ! -x `which gh` ]; then
+    echo "You must have github's gh client installed to install a preview version"
+    exit 1
+  fi
+
+  IFS=/ read -r owner repo << EOF
+${REPO}
+EOF
+
+  JQCMD='.data.repository.ref.target.history.nodes | map(select(.statusCheckRollup.state == "SUCCESS")) | first | .oid'
+  PREVIEW_COMMIT=`gh api graphql  -f owner=${owner} -f repo=${repo} -f branch=${BRANCH} --jq "${JQCMD}" -F query='
+query($owner: String!, $repo: String!, $branch: String!) {
+  repository(owner: $owner, name: $repo) {
+    ref(qualifiedName: $branch) {
+      target {
+        ... on Commit {
+          history(first: 10) {
+            nodes {
+              oid
+              statusCheckRollup {
+                state
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+'`
+
+  if [ "$REPO" = "kong/kong-mesh" ]; then
+    PREVIEW_COMMIT=$(echo $PREVIEW_COMMIT | cut -c -7)
+  else
+    PREVIEW_COMMIT=$(echo $PREVIEW_COMMIT | cut -c -9)
+  fi
```

Comment:
Please do the change to always use 9 I'll open the PR to fix the conf today

Reply:
It's here: https://github.com/kumahq/kuma/pull/5296

Reply:
Done.